README.md

```diff
@@ -31,12 +31,12 @@ Add the following lines to your Podfile:
 
 ```ruby
 # Required
-pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :commit => 'c5a637a201a3043c24e52f3e2c50a1657a8ffced'
-pod 'MapboxNavigationUI.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :commit => 'c5a637a201a3043c24e52f3e2c50a1657a8ffced'
+pod 'MapboxNavigation.swift', '~> 0.1.0'
```

Comment:
This style of `pod` depends on CocoaPods.org, so make sure you `pod trunk push` both these podspecs right after release.

While you’re at it, you might want to consider making one of these podspecs into a [subspec](https://guides.cocoapods.org/syntax/podspec.html#subspec) of the other to avoid fragmenting the user base.